### DISCUSSION TITLE
Punctuation and Capitalization Model for other languages?

### DISCUSSION THREAD

**Discussion by vitotitto** [NVIDIA-NeMo/NeMo#3640] [Q&A]
Hi guys, I want to analyse some transcripts that miss punctuation. It looks like the lack of it distorts the results. 
I found the page https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/punctuation_and_capitalization.html and the "punctuation_en_bert" model works great for English text to get punctuation back. 
I was trying to find models for other languages and wanted to confirm these do not exist yet?

**Comment by ekmb** [comment]
@vitotitto, there are no pre-trained punctuation models available for other languages at the moment.
